docs/source/_src_user_guide/state_propagation/propagation_setup/propagation_architecture.rst

```diff
@@ -0,0 +1,81 @@
+* If the ``set_integrated_result`` to the :class:`~tudatpy.numerical_simulation.SingleArcSimulator` is set to true, the propagated states (in conventional formulation) are used to reset the environment of the propagated body/bodies. For the different state types, this means:
+
+  * Translational dynamics: the propagated translational state of the body is used to create an interpolator (:func:`~tudatpy.math.interpolators.lagrange_interpolation`, ``number_of_points``=6), which is used to update the :func:`~tudatpy.numerical_simulation.environment_setup.ephemeris.tabulated` ephemeris of the body. If needed, a translation from the propagation origin to the ephemeris origin is applied (see :ref:`translational_frame_origins`). NOTE: this is *only* possible if the body has a tabulated ephemeris alreacy, or no ephemeris. In the latter case a tabulated ephemeris is created, with ephemeris origin equal to the propagation origin. In case you want to use a non-tabulated ephemeris for the propagated body, you can use the :func:`~tudatpy.numerical_simulation.environment_setup.ephemeris.tabulated_from_existing` function to override existing body settings (see :ref:`override_body_settings`). When doing so, the behaviour of the non-tabulated ephemeris will be emulated by a non-tabulated ephemeris.
+  *  Rotational dynamics: the propagated rotational state of the body is used to create an interpolator (:func:`~tudatpy.math.interpolators.lagrange_interpolation`, ``number_of_points``= 6), which is used to create a tabulated rotation model (not yet exposed to Python). At present, this option is only possible if the propagated body starts out with *no* rotation model. An update to allow the same flexibility as for the translational dynamics (see above) is planned
+  *  Mass dynamics: the propagated mass of the body is used to create an interpolator (:func:`~tudatpy.math.interpolators.lagrange_interpolation`, ``number_of_points``=6), which is used to update the mass function of the body.
+* If the ``clear_numerical_solutions`` to the :class:`~tudatpy.numerical_simulation.SingleArcSimulator` is set to true, the state (processed and unprocessed) and dependent variable history are deleted, *after* having reset the environment (if ``set_integrated_result`` was set to true; see above). In this case, the ephemerides are reset with the propagated dynamics, but the the results of the propagation cannot be extracted from the :attr:`~tudatpy.numerical_simulation.SingleArcSimulator.unprocessed_state_history`, :attr:`~tudatpy.numerical_simulation.SingleArcSimulator.state_history` and :attr:`~tudatpy.numerical_simulation.SingleArcSimulator.dependent_variable_history` attributes. Note that the dependent variable history will be lost entirely in this case.
```

Comment:
I am not sure about TudatPy but it might be worth mentioning (here or elsewhere) that, for Tudat, clear_numerical_solutions is by default set to true for multi-arc propagation. That could also go in the "Point of attention/specific considerations" page for propagation?